ShortcutTrainerBackend/ShortcutTrainerBackend/Data/Models/CourseParameter.cs

```diff
@@ -2,10 +2,10 @@
 {
     public class CourseParameter
```

Comment:
Die Klasse CourseParameter ist obsolet. Die Parameter in den Controllern sollen einzeln ausgelistet werden und je nachdem ob notwendig oder nicht, sind sie entweder nicht nullable oder nullable